etc/env.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
 source "/scratch3/BMC/gsd-hpcs/Daniel.Abdi/miniconda3/etc/profile.d/conda.sh"
 conda activate hrrrcast-live
+pip install eccodes
+pip install scitools-iris
+pip install iris-grib
```

Comment:
These are not needed for Ursa, since they are installed via the environment. Also get an error message, for each job when trying to install them every time.

Reply:
Removed